en/02_Developer_Guides/01_Templates/01_Syntax.md

```diff
@@ -89,18 +88,16 @@ These variables will call a method / field on the object and insert the returned
 - `$Foo(param)` will call `$obj->Foo("param")`
 - `$Foo.Bar` will call `$obj->Foo()->Bar()`
 
+> [!WARNING]
+> If you wish to pass parameters to getter functions, you must use the full method name, e.g. $getThing('param'). Also, parameters must be literals, and cannot be other template variables (`$getThing($variable)` will not work)
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.